mod/core/entrypoint.sh

```diff
@@ -15,6 +15,8 @@ mkdir -p /var/bigbluebutton/published
 mkdir -p /var/bigbluebutton/deleted
 mkdir -p /var/bigbluebutton/unpublished
 
+# -- fix directory permissions
+RUN chown bigbluebutton:bigbluebutton /var/bigbluebutton
```

Comment:
For me, this commit here breaks the upload presentation functionality: Uploading a file hangs and I get an error message
```
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]: Exception in thread "testpage.pdf" java.lang.NullPointerException
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.bigbluebutton.presentation.SupportedDocumentFilter.isSupported(SupportedDocumentFilter.java:42)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.bigbluebutton.presentation.DocumentConversionServiceImp.processDocument(DocumentConversionServiceImp.java:59)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.bigbluebutton.presentation.DocumentConversionService$processDocument.call(Unknown Source)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.bigbluebutton.web.services.PresentationService$_closure5$_closure16.doCall(PresentationService.groovy:85)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.bigbluebutton.web.services.PresentationService$_closure5$_closure16.doCall(PresentationService.groovy)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at java.lang.reflect.Method.invoke(Method.java:498)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:98)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:264)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1034)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at groovy.lang.Closure.call(Closure.java:418)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at groovy.lang.Closure.call(Closure.java:412)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at org.codehaus.groovy.runtime.DefaultGroovyMethods$2.run(DefaultGroovyMethods.java:15553)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at java.util.TimerThread.mainLoop(Timer.java:555)
core_1        | May 17 06:25:01 v220200423890112953 run-prod.sh[61]:         at java.util.TimerThread.run(Timer.java:505)
```

Indeed,
```
root@v220200423890112953:/var/bigbluebutton# ls -ld /var/bigbluebutton/
drwxr-xr-x 9 red5 red5 4096 May 17 06:06 /var/bigbluebutton/
```
belongs to `red5`, but most of its contents do not:
```
root@v220200423890112953:/var/bigbluebutton# ls -l /var/bigbluebutton/
total 28
drwxr-xr-x 2 root root 4096 May  2 03:28 configs
drwxr-xr-x 2 root root 4096 May 17 06:06 deleted
drwxr-xr-x 2 root root 4096 May  2 03:32 diagnostics
drwxr-xr-x 2 root root 4096 May 17 06:06 published
drwxr-xr-x 6 root root 4096 May 17 06:06 recording
drwxr-xr-x 2 red5 red5 4096 Mar 25 20:43 screenshare
drwxr-xr-x 2 root root 4096 May 17 06:06 unpublished
```
Executing
```
chown -R bigbluebutton.bigbluebutton /var/bigbluebutton
```
in the running container then allows to upload presentations.

Reply:
the wrong ownership breaks the upload presentation functionality, this commit should prevent this error, but apparently applying it only to the folder itself isn't enough. I added now the recursive statement.
thanks for the report!